Fix otel log payload

Danwakeem opened the pull request:
## Description
The format of the resources/attributes was not correct so this PR just just adjusting the format of the output 👍

Danwakeem committed:
Fixed otel log payload format

Danwakeem committed:
Fixed otel log payload format

Danwakeem committed:
Bump version

medikoo reviewed (commented):
Can we back with some regression test?

Danwakeem committed:
Added log test

Danwakeem commented:
@medikoo No problem I added a log test here 😎 Thanks for the tip on adding that test 👏 

There is one kind of odd thing I did here which was add an environment variable called `SLS_TEST_PRINT_LOG_EVENT`. I added this because the current version of the serverless cli will not add a `SLS_OTEL_REPORT_LOGS_URL` yet and I didn't want this to do a console.log in a live serverless app 👍 

Let me know if you think I should change that in some way.

medikoo reviewed (approved):
Looks great 👍
